drivers/spi/spi_ll_stm32.c

```diff
@@ -338,20 +338,57 @@ static int spi_stm32_get_err(SPI_TypeDef *spi)
 	return 0;
 }
 
-/* Shift a SPI frame as master. */
-static void spi_stm32_shift_m(SPI_TypeDef *spi, struct spi_stm32_data *data)
+static int spi_stm32_can_use_fifo(SPI_TypeDef *spi,
+								  struct spi_stm32_data *data)
```

Comment:
Nit: indentation

Reply:
I tried to indent this better at first: by using a tab at the beginning of the line and then all spaces until it gets aligned with the `spi` parameter. The indentation checks threw an error. 

Another solution is to add a couple of tabs instead of aligning it with the `spi` parameter. I used to do this until I was told that parameters should be aligned. I believe that comment is in this PR.

What is the actual criteria to be followed?

Reply:
You should use 8 chars-tabs and complete with spaces for the last 7 indentation chars.

Reply:
I believe the indentation is correct now.